src/app/clusters/tls-certificate-management-server/tls-certificate-management-server.h

```diff
@@ -34,7 +34,9 @@ namespace Clusters {
 
 class TlsCertificateManagementDelegate;
 
-class TlsCertificateManagementServer : private AttributeAccessInterface, private CommandHandlerInterface
+class TlsCertificateManagementServer : private AttributeAccessInterface,
+                                       private CommandHandlerInterface,
+                                       private chip::FabricTable::Delegate
```

Comment:
```suggestion
                                       private FabricTable::Delegate
```
Already in `namespace chip`?